src/DataTable/DataTableLayout.jsx

```diff
@@ -27,11 +28,13 @@ const DataTableLayout = ({
 
 DataTableLayout.defaultProps = {
   className: null,
+  filtersTitle: undefined,
 };
 
 DataTableLayout.propTypes = {
   className: PropTypes.string,
   children: PropTypes.node.isRequired,
+  filtersTitle: PropTypes.string,
```

Comment:
This prop should likely accept things beyond strings (either node or element maybe?) , e.g. say a consumer wants to wrap the title in a custom element with a class name, (e.g., `filtersTitle={<span className="...">{...}</span>}`).

Reply:
Yeah, it definitely should, missed it, thanks